flareio/api_client.py

```diff
@@ -24,7 +26,31 @@ def __init__(
 
         self._api_token: t.Optional[str] = None
         self._api_token_exp: t.Optional[datetime] = None
-        self._session = session or requests.Session()
+        self._session = session or self._create_session()
+
+    @staticmethod
+    def _create_session() -> requests.Session:
+        session = requests.Session()
+        retries = Retry(
+            total=5,
+            backoff_factor=1,
+            status_forcelist=[429, 502, 503, 504],
+            allowed_methods={"GET", "POST"},
+            backoff_max=15,
+        )
+        session.mount(
```

Comment:
Why mount for `http://` ?

Reply:
testing stuff...

Reply:
We can't really test it. So I think i'll remove the unit test and just merge and hope for the best